src/cli/util.js

```diff
@@ -597,135 +605,11 @@ function groupBy(array, getKey) {
   }, Object.create(null));
 }
 
-/** @param {'api' | 'cli'} type */
-function normalizeConfig(type, rawConfig, options) {
-  if (type === "api" && rawConfig === null) {
-    return null;
-  }
-
-  options = options || {};
-
-  const consoleWarn =
-    options.warning === false ? () => {} : logger.warn.bind(logger);
-
-  const normalized = {};
-
-  Object.keys(rawConfig).forEach(rawKey => {
-    const rawValue = rawConfig[rawKey];
-
-    const key = type === "cli" ? rawKey : dashify(rawKey);
-
-    if (type === "cli" && key === "_") {
-      normalized[rawKey] = rawValue;
-      return;
-    }
-
-    if (type === "cli" && key.length === 1) {
-      // do nothing with alias
-      return;
-    }
-
-    let option = constant.detailedOptionMap[key];
-    if (type === "api" && option === undefined) {
-      option = constant.apiDetailedOptionMap[key];
```

Comment:
This was the only consumer for `apiDetailedOptionMap`, so probably just remove it entirely. This only dates back to https://github.com/prettier/prettier/pull/3584. It's not part of the public API and never made it into a release anyway.

Reply:
Oops, misread. Never mind. Sorry.